packages/components/Callout/package.json

```diff
@@ -40,6 +40,7 @@
   "devDependencies": {
     "@fluentui-react-native/eslint-config-rules": "^0.1.1",
     "@fluentui-react-native/scripts": "^0.1.1",
+    "@office-iss/react-native-win32": "^0.68.0",
```

Comment:
We should probably add React Native macOS too. And maybe to peer dependencies.

Reply:
The RN Win32 is specifically needed for the jest test, not for any runtime code. Is adding RN macOS and adding peer dependencies necessary for that?

Reply:
Fair enough. I'll resolve the comments.